nucleus/dataset_item.py

```diff
@@ -89,6 +114,58 @@ class DatasetItemType(Enum):
 
 @dataclass  # pylint: disable=R0902
 class DatasetItem:  # pylint: disable=R0902
+    """A dataset item is an image or pointcloud that has associated metadata.
+
+    Note: for 3D data, please include a :class:`.CameraParams` object under a key named
+    "camera_params" within the metadata dictionary. This will allow for projecting
+    3D annotations to any image within a scene.
+
+
+    Attributes:
+        image_location: (required if pointcloud_location not present) The location
```

Comment:
If you check the rendered docs, "(required if pointcloud_location ...)" shows up kind of weird :/ not really sure of a better way to show this. It's fine for now ofc just something to think about

Reply:
Tried something new